src/index.ts

```diff
@@ -11,7 +11,6 @@ const server = new ApolloServer({
 const port = 3001;
 
 server.listen({ port }).then(({ url }) => {
+    // eslint-disable-next-line no-console
     console.log(`🚀  Server ready at ${url}`);
 });
-
-// add some lint
```

Comment:
I can't comment on the mockData file, but we should keep this. The main use case is for running automation tests.

Reply:
yep! ha, I think our first step is to start having some nice integration tests that actually use it!